server/db/models/Organization.js

```diff
@@ -5,6 +5,49 @@ module.exports = function (sequelize, DataTypes) {
   }, {
     classMethods: {
 
+      createOrganization: function (name, subscription) {
+        return Organization.create({
+          name: name,
+          subscription: subscription
+        });
+      },
+
+      getOrganizationInfo: function (id) {
+        return Organization.find({
+          where: {
+            id: id
+          }
+        });
+      },
+
+      updateOrganizationInfo: function (id, name, subscription) {
```

Comment:
should we just pass in params here instead of name and subscription? that way we won't need to update this method if we add more stuff to the schema for organization